shared/xccdf/system/accounts/physical.xml

```diff
@@ -252,6 +252,40 @@ access when the system is rebooted.
 <!--ref nist="IA-2(1),AC-3" disa="213" /-->
 </Rule>
 
+<Rule id="disable_ctrlaltdel_ctrlaltdelburstaction" severity="high" prodtype="rhel7">
+<title>Disable Ctrl-Alt-Del CtrlAltDelBurstAction</title>
+<description>
+By default, <tt>SystemD</tt> will reboot the system if the <tt>Ctrl-Alt-Del</tt>
+key sequence is pressed Ctrl-Alt-Delete more than 7 times in 2s.
+<br /><br />
+To configure the system to ignore the <tt>Ctrl-Alt-Del CtrlAltDelBurstAction</tt>
+setting modify <tt>/etc/systemd/system.conf</tt>:
+<pre>CtrlAltDelBurstAction=none</pre>
+</description>
+<ocil clause="the system is configured to reboot when Ctrl-Alt-Del is pressed
+more than 7 times in 2s.">
+To ensure the system is configured to ignore the Ctrl-Alt-Del setting,
+enter the following command:
+<pre>$ sudo sed -i 's/^#CtrlAltDelBurstAction=.*/CtrlAltDelBurstAction=none/' /etc/systemd/system.conf</pre>
+</ocil>
+<rationale>
+A locally logged-in user who presses Ctrl-Alt-Del, when at the console,
+can reboot the system. If accidentally pressed, as could happen in
+the case of mixed OS environment, this can create the risk of short-term
+loss of availability of systems due to unintentional reboot.
+</rationale>
+<warning category="general">Disabling the <tt>Ctrl-Alt-Del</tt> key sequence
+with <tt>SystemD</tt> DOES NOT disable the <tt>Ctrl-Alt-Del</tt> key sequence
+if running in <tt>graphical.target</tt> mode (e.g. in GNOME, KDE, etc.)! The
+<tt>Ctrl-Alt-Del</tt> key sequence will only be disabled if running in
+the non-graphical <tt>multi-user.target</tt> mode.
+</warning>
+<oval id="disable_ctrlaltdel_ctrlaltdelburstaction" />
+<ident prodtype="rhel7" cce="xxxx" />
+<ref prodtype="rhel7" stigid="xxxx" />
+<ref nist="xxxx" disa="xxxx" ossrg="xxxx" cui="xxxx" />
```

Comment:
This can be:
`<ref nist="AC-6" disa="366" ossrg="SRG-OS-000480-GPOS-00227" cui="3.4.5" />`